docs/maintainers/maintainers-guide.md

```diff
@@ -142,7 +142,7 @@ Highlights can be used to:
 
 There are two ways to create a new highlight:
 
-1. After clicking the [Highlights tab](https://app.opensauced.pizza/feed) in the top navigation, click the input that says "Post a highlight to show your work!"
+1. After clicking the "[Highlights](https://app.opensauced.pizza/feed)" tab in the top navigation, click the input that says "Post a highlight to show your work!"
 
 2. From your profile, click the input that says "Post a highlight to show your work!"
```

Comment:
[suggestion] We mentioned this part in the previous step, so I think it is best to omit this part. Thoughts?
```suggestion
```